assets/css/custom.scss

```
@@ -3908,3 +3908,1394 @@ font-size: 1.55rem;
+$cs-pine-deep: #0a2a2c;
+$cs-pine-soft: #13474a;
+$cs-pine-line: #1f5a5d;
+$cs-mint-soft: rgba(42, 217, 194, 0.12);
+$cs-mint-faint: rgba(42, 217, 194, 0.06);
+
+@mixin cs-brand-gradient {
+  background: linear-gradient(92deg, #ede497 -0.66%, #2ad9c2 26%, #d891ce 59.55%);
+}
+@mixin cs-brand-gradient-text {
+  @include cs-brand-gradient;
+  -webkit-background-clip: text;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)